llvmlite/ir/instructions.py

```diff
@@ -676,3 +676,28 @@ def descr(self, buf):
                               clauses=''.join(["\n      {0}".format(clause)
                                                for clause in self.clauses]),
                               ))
+
+class Fence(Instruction):
+    """
+    The `fence` instruction.
+
+    As of LLVM 5.0.1:
```

Comment:
Thanks for the patch. Will this be valid under LLVM 6.0 too?

Reply:
The syntax for `fence` changed from [LLVM 4](https://releases.llvm.org/4.0.0/docs/LangRef.html#fence-instruction) to [LLVM 5](https://releases.llvm.org/5.0.0/docs/LangRef.html#fence-instruction), which is why I mentioned it, but since then there have been no changes in [LLVM 6](https://releases.llvm.org/6.0.0/docs/LangRef.html#fence-instruction) nor in [LLVM trunk](https://www.llvm.org/docs/LangRef.html#fence-instruction).

Reply:
Ah great, thanks for confirming.